test/jdk/java/net/DatagramSocket/SendDatagramToBadAddress.java

```diff
@@ -170,6 +175,8 @@ private void test(DatagramSocket sock) throws Exception {
                 sock.send(p);
                 p = new DatagramPacket(buf, buf.length, addr, port);
                 sock.receive (p);
+                print("(unexpectedly) received data from address " + p.getAddress()
```

Comment:
We should immediately fail the test here

Reply:
Hello Daniel,
I had considered that when doing this change. But this test itself appears to be very old one and I don't have any historical context of why it is doing these checks in a loop. Furthermore, even after it receives an exception (either a socket timeout one or any general exception), after setting a flag in case of a general exception, the test continues to run the next iteration of the loop. So I didn't want to change that logic, at least not this PR whose aim is to just enable debug logging.
However, if others too think we should fail here immediately, I'll go ahead do that change.